src/main/java/za/ac/cput/utility/Helper.java

```diff
@@ -0,0 +1,19 @@
+package za.ac.cput.utility;
+
+/*
+author: Ameer Ismail
+student nr: 218216033
+Utility: Helper class
+ADP 3 Assignment Group1
+*/
+
+
+import java.util.UUID;
+
+public class Helper
+{
+    public static String generateId()
```

Comment:
How do you feel about removing the generateId method?
I saw that its not being used anywhere.